bundles/org.openhab.binding.insteon/README.md

```diff
@@ -131,6 +132,7 @@ In order to determine which channels a device supports, you can look at the devi
 | broadcastOnOff | Switch | Broadcast On/Off |
 | contact | Contact | Contact |
 | coolSetPoint | Number | Cool Set Point |
+| dawndusk | Contact | Light Sensor above or below threshold |
```

Comment:
Can we use something more generic than "dawndusk"? This assumes that the variable is associated with the sun rise/set

Reply:
I can make it whatever you'd like, but I think this is the traditional way to refer to this feature on motion sensors going back to old X10 stuff.  How about lightdark?

Reply:
changed to lightdark

Reply:
This is a contact so it should be something that describes what it actually is doing and needs to be camel case (lightDark). Can you look at other bindings and see if you can find a generic term?

Reply:
So I grepped around in the Zwave and Zigbee bindings and I don't see any other motion sensors with this feature.  Many report light - similar to this motion sensor, but none seem to report when the light is over or under the configurable threshold.  I pushed a new version changing to camel case as you asked but left it as lightDark.  I'm open to changing it to to whatever, just not sure what else it better.

Reply:
Since this is a contact, lets use something that describes what the contact is telling us. How about `lightLevelAboveThreshold`? Then change the line to:

| lightLevelAboveThreshold| Contact | Light Level Above/Below Threshold |